TERMINOLOGY.md

```diff
@@ -0,0 +1,53 @@
+# Terminology
+
+## Estimators and Predictors
+Fairlearn largely follows the [terminology established by scikit-learn](https://scikit-learn.org/stable/developers/contributing.html#different-objects), specifically:
+- Estimator: implements a `fit` method
+- Predictor: implements a `predict` method
+However, contrary to [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator) we do not require determinism. Lots of unfairness mitigation techniques require randomized output to meet their objective in expectation. We do provide unrandomized versions of every method as well, but they may not be able to uphold the same theoretical guarantees.
+
+## Randomized Predictors
+
+Rather than providing deterministic output many predictors produced by unfairness mitigation techniques in the fairness literature are randomized. As a consequence, it is possible to get different output from the predictor's `predict` method when providing samples with identical features. `predict_proba` provides insights on the probabilities themselves. If randomized predictions are not allowed or undesirable, it is possible to always use the most likely prediction from `predict_proba`. Please be aware that this can affect the fairness of the resulting predictions.
```

Comment:
I thought it was 'predict_posterior' which was going to give the individual predictions from each model (along with the probability of that model being chosen)?

Reply:
Depends on how you look at it, but I agree that this is not specific enough, so I'll take that part out. We should write about what to expect from predict, predict_proba, etc. soon though